grid_helper.js

```diff
@@ -436,17 +442,65 @@ function buildView(divId, initialLayout, initialContent=[], numCols=12, gridRowH
           window.location.hash = that.locationHash;
           that.currentTableauName = name;
         },
-        isCurrentTableau: function(name) {
-          return this.currentTableauName === name;
+        isCurrentTableau: function(tableauName) {
```

Comment:
It appears that with modern JS, we can write this:
```javascript
isCurrentTableau: function(tableauName) {
  return this.currentTableauName === tableauName;
},
```

as the more concise:
```javascript
isCurrentTableau(tableauName) {
  return this.currentTableauName === tableauName;
},
```

In my latest PR, I adjusted all of the functions in the Vue controller accordingly.